.devcontainer/conda-forge/devcontainer.json

```
@@ -0,0 +1,41 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+{
+    "name": "Miniforge (default-channel=conda-forge)",
+    "image": "condaforge/miniforge3:latest",
+
+    // Features to add to the dev container. More info: https://containers.dev/features.
+    // "features": {},
+
+    // Use 'forwardPorts' to make a list of ports inside the container available locally.
+    // "forwardPorts": [],
+    "mounts": [
+        "source=${localWorkspaceFolder}/../conda,target=/workspaces/conda,type=bind,consistency=cached",
+        "source=${localWorkspaceFolder}/../mamba,target=/workspaces/mamba,type=bind,consistency=cached"
+    ],
+    // Use 'postCreateCommand' to run commands after the container is created.
+    "postCreateCommand": "bash /workspaces/conda-libmamba-solver/.devcontainer/post_create.sh",
+    // Use 'postStartCommand' to run commands after the container is started.
+    "postStartCommand": "bash /workspaces/conda-libmamba-solver/.devcontainer/post_start.sh",
+
+    // Configure tool-specific properties.
+    "customizations": {
+        "vscode": {
+            "settings": {
```

```suggestion
            "settings": {
                "python.defaultInterpreterPath": "/opt/conda/bin/python",
```
We should add the default python interpreter as well, cf https://github.com/devcontainers/images/blob/a8e1c3a68500f34bb6332d6ad981f9e29d72f0c9/src/miniconda/.devcontainer/devcontainer.json#L20